cpp/include/cudf/io/parquet.hpp

```diff
@@ -39,9 +39,10 @@ namespace io {
  */
 
 constexpr size_t default_row_group_size_bytes   = 128 * 1024 * 1024;  ///< 128MB per row group
-constexpr size_type default_row_group_size_rows = 1000000;     ///< 1 million rows per row group
-constexpr size_t default_max_page_size_bytes    = 512 * 1024;  ///< 512KB per page
-constexpr size_type default_max_page_size_rows  = 20000;       ///< 20k rows per page
+constexpr size_type default_row_group_size_rows = 1000000;      ///< 1 million rows per row group
+constexpr size_t default_max_page_size_bytes    = 512 * 1024;   ///< 512KB per page
+constexpr size_type default_max_page_size_rows  = 20000;        ///< 20k rows per page
+constexpr size_type default_column_index_truncate_length = 64;  ///< truncate to 64 bytes
```

Comment:
Typically, names of constants should be ALL_CAP.

Reply:
Yes, I normally would do all caps, but was following the convention already established in the file.  Should I change them all to caps?

Reply:
Hmm. I don't know, I've received mixed feedback on this during reviews. I'd personally use ALL_CAPS, but given that you're prefixing these with `default_`, I think it's legible as is.

Reply:
You may have a look at my comment below. The code snippet associated to it has an ALL_CAP constant.

Reply:
Not trying to be difficult, but from the [Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-all-caps): "Use ALL_CAPS for macro names only".  Maybe this is a larger conversation to have outside this PR.

Reply:
This seems to have not been enforced in cudf yet. But it has already been used, as you can seen the ALL_CAP constants in many places.

In the past, when there was not `constexpr`, the typical way to create a compile-time constants is to use macros. So, ALL_CAP constant-macros have been used. Nowaday, we have `constexpr` variables to replace constant-macros.  However, we should, by name convention, have a way to differentiate the typical variables and compile-time constants. Using ALL_CAP name for `constexpr` variables  is the best solution so far.

Reply:
Yes, it's not enforced and the libcudf code has examples of both styles (and yes, I'm inconsistent about it as well...my brain wants constants ALL_CAPS too :smile:). It seems the C++ community in general is gravitating away from ALL_CAP since that leads to confusion about whether something is a variable or a macro.  All I want is some kind of team consensus, that moving forward either ALL_CAPS, or snake case, or kCamelCase will be used for `constexpr` values.  I'm not on the team, so I don't get a vote :wink:

Reply:
We can fix that if you'd like...